### DISCUSSION TITLE
How to call the interface circularly

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#1242] [Questions and Help] [answered]
Hi， I now make an application with lowdefy, select several users and call the interface to pass the user information to the back end. The selected user is an array. I need to call the interface circularly. What should I do?

**Comment by SamTolmay** [comment, marked as answer]
Hi @xiaoxiji1234 

There isn't a feature to do this in Lowdefy currently, so you will need to use a custom JsAction. The JsAction receives a `context` object as it's first argument, which has functions like `Request` and `SetState` which you can use to interface with other Lowdefy config. 

I know another user has had success with this approach, but unfortunatly I don't have an example I can share with you.

**Reply by xiaoxiji1234** [reply]
Ok, I got it. Thank you!
